scripts/all-core.sh

```diff
@@ -157,14 +157,22 @@ pre_initialize_variables () {
             CRYPTO_CONFIG_H='tf-psa-crypto/include/psa/crypto_config.h'
             PSA_CORE_PATH='tf-psa-crypto/core'
             BUILTIN_SRC_PATH='tf-psa-crypto/drivers/builtin/src'
-            CONFIG_TEST_DRIVER_H='tf-psa-crypto/tests/configs/config_test_driver.h'
+            if in_3_6_branch; then
```

Comment:
I feel that the branch detection logic here is redudant, but I added it for now.